packages/@finetwork:ui/src/components/Input/index.tsx

```diff
@@ -59,6 +59,7 @@ export const Input: InputComponent = React.forwardRef(
       containerProps = {},
       error,
       info,
+      borderColor = '#000',
```

Comment:
Why add `borderColor` prop and not with `compundVariants` ?

Reply:
because if I use variants I have to create a variant for each color I want to use. This way I can specify the every color I want

Reply:
What is the goal to set a borderColor ? borderColor is already declared in styled, so why is the main goal to override that ?

Reply:
I mean, with this code, you cannot override the `borderColor` or even any css prop because you are setting `css` prop.

Reply:
I menainput has a div inside that I can't modify using compoundVariants and when it's disabled, automatically the border color is gray and can't be modified

Reply:
I can override the `borderColor` because I'm setting a custom border color using `css` and not only: 'primary', 'secondary', 'tertiary', 'disabled'

Reply:
But there is a `containerProps` prop.

Reply:
How could I use that to change `borderColor`? Because I cannot desclare `css={{borderColor:'#000'}}` inside `containerProps`

**My option**
`
<Input disabled={true} defaultVale="Hello World" borderColor="$primaryText" />
`

**Your option**
`
<Input disabled={true} defaultVale="Hello World" containerProps={?} />
`

Reply:
Sorry, I was wrong, it's the `inputContainerProps` prop.
You need to do this:
```jsx
<Input disabled={true} defaultVale="Hello World" inputContainerProps={{css: {borderColor: 'yellow'}}} />
```
Here's an example
https://codesandbox.io/s/input-tag-finetwork-ui-9j5mg8?file=/src/App.js

Reply:
I consider my option easier to undertand and use

Reply:
It's ok, but this is a breaking change, with this change it's impossible to override styles in the `StyledContainerInput` component, so be careful that it doesn't affect in some implementation.

Reply:
I understant now. What do you think about this solution:

`css={{ borderColor, ...inputContainerProps.css }}`

Reply:
That would be OK, but it's necessary to initiate `css` prop inside `inputContainerProps`.
What about to initiate `inputContainerProps` with `css` and `borderColor` ?

```js
inputContainerProps = { css: { borderColor: '#000' } }
```

In that way the `borderColor` prop wouldn't be necessary.